tests/rados/scheduled_scrub_scenarios.py

```diff
@@ -185,4 +183,7 @@ def run(ceph_cluster, **kw):
         sys.stderr.write("ERRORED LINE\t::%s:\n" % str(exc_tb.tb_lineno))
         return 1
     finally:
-        set_default_params(rados_obj)
+        remove_parameter_configuration(mon_obj)
+        rados_obj.set_osd_flags("unset", "noscrub")
+        rados_obj.set_osd_flags("unset", "nodeep-scrub")
```

Comment:
The logs suggest that both the commands' execution has failed, this is happening because output of the command execution is present in ERR stream and the way that `set_osd_flags` https://github.com/red-hat-storage/cephci/blob/e6500e8d7b0c7024e48a9eda02279733633a4099/ceph/rados/rados_scrub.py#L164 has been designed, it is assuming the execution failed if error stream has any content.
Discussed the fix offline, awaiting changes in `set_osd_flags` method